Source/ExpressionComparer.cs

```diff
@@ -40,14 +40,12 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq.Expressions;
 using System.Collections.ObjectModel;
-using System.Reflection;
-using System.Linq;
+using System.Linq.Expressions;
 
-namespace Moq.Matchers
+namespace Moq
```

Comment:
We no longer use `ExpressionComparer` just together with matchers, so it's moved into the main namespace.